src/main/webapp/app/courses/results/result.component.ts

```diff
@@ -60,17 +51,18 @@ export class ResultComponent implements OnInit, OnChanges, OnDestroy {
             this.results = this.participation.results;
 
             this.init();
-            // make sure result and participation are connected
+            // Make sure result and participation are connected
             if (this.result) {
                 this.result.participation = this.participation;
             }
 
+            // Initial refresh call
+            this.refresh(false);
```

Comment:
Are you sure that this code does not introduce breaking changes in other components, e.g. the courses dashboard or the exercise dashboard? I think this code is used in these places as well, so please make 100% sure that it does not introduce problems in the other places!